pytapo/__init__.py

```diff
@@ -1902,6 +1858,27 @@ def getMost(self, omit_methods=[]):
                 return self.getMost(["getAudioConfig"])
             else:
                 raise Exception(f"Unexpected camera response: {results}")
+        
+        returnData = {}
+
+        for request in requestData["params"]["requests"]:
+            returnData[request["method"]] = False
+
+        for result in results["result"]["responses"]:
+            if (
+                "error_code" in result and result["error_code"] == 0
+            ) and "result" in result:
+                if result["method"] not in returnData or (returnData[result["method"]] is False):
+                    returnData[result["method"]] = result["result"]
+                else:
+                    raise Exception(
+                        f"Unexpected two valid responses for same method. Method: {result['method']} Responses: {returnData[result['method']]} and {result['result']}"
+                    )
```

Comment:
Could we make it a list of results instead for cases where the device responds to both methods? HA can then loop through the results and find the one it is looking for.

Reply:
there is not going to be a way to know which one is first and second anyway, because can be in different order, and it would force tapo-control to always check if it is a dictionary or a list, are you sure that is worth making it a list? for what we have seen there are no devices that respond to both, if some day appears, there should be some extra research to understand why it responds to both, its an strange case to have two sirens on the same device